auditing/auditing-interceptor.go

```diff
@@ -323,9 +325,10 @@ func (i auditingConnectInterceptor) WrapUnary(next connect.UnaryFunc) connect.Un
 	}
 }
 
-func NewConnectInterceptor(a Auditing, logger *zap.SugaredLogger, shouldAudit func(fullMethod string) bool) connect.Interceptor {
+func NewConnectInterceptor(a Auditing, logger *slog.Logger, shouldAudit func(fullMethod string) bool) connect.Interceptor {
```

Comment:
Would be great to change the signature so the function returns an error so the caller has to handle the case if the `a` parameter is nil.

Reply:
done